src/newbank/server/model/Transaction.java

```diff
@@ -0,0 +1,69 @@
+package newbank.server.model;
+
+import java.util.Calendar;
+import java.util.Date;
```

Comment:
Hi @kelinj ,

Please try to use the API from java.time.* package instead of the java.util.*

Reply:
Made the changes. I didn't know java.util.Calendar and Date are outdated. Came to know only after doing some research now on this. Thanks for the info